lib/rubocop/cop/rails/active_record_aliases.rb

```diff
@@ -0,0 +1,45 @@
+# frozen_string_literal: true
+
+module RuboCop
+  module Cop
+    module Rails
+      # Finds usage of `update_attributes`/`update_attributes!`. Prefer
+      # the unaliased `update`/`update!`.
```

Comment:
This cop's name is an abstract name for related to aliases of Active Record. If you'd like to keep this cop name, it may be better to describe it similarly to the similarly named `Rails/ActiveSupportAliases` cop.
https://github.com/bbatsov/rubocop/blob/v0.52.1/lib/rubocop/cop/rails/active_support_aliases.rb#L6-L7